department-of-veterans-affairs/caseflow
# Paula/caseflow 230 add info success message

Resolves [CASEFLOW-230](https://vajira.max.gov/browse/CASEFLOW-230)

### Description
When a user updates the NOD date successfully, the success alert message has been updated to say "The NOD date for apellantName's case has been updated from oldDate(MM/YY/YYYY) to newDate(MM/YY/YYYY). There have been no changes to timeliness of issues."  

### Acceptance Criteria
- [ ] Alert message has been updated to the new message

### Testing Plan
1. Login to Caseflow as a user that has permissions to edit the NOD date in Queue (ex:  BVAAABSHIRE)
2. Select an AMA case that has an NOD.
3. Edit the NOD date.
4. Confirm that the success alert message includes the Appellant's name and the old and new dates.

### User Facing Changes
 
<img width="681" alt="NOD date success alert update" src="https://user-images.githubusercontent.com/23639439/103907032-baa82980-50ce-11eb-9c08-9533212f5ca4.png">
